doc/src/explanation/active-deprecations.md

```diff
@@ -76,11 +76,21 @@ SymPy deprecation warnings.
 
 ## Version 1.12
 
+(deprecated-mechanics-joint-coordinate-format)=
+### New Joint coordinate format
+The format of the generalized coordinates and generalized speeds of the joints
+in the ``sympy.physics.mechanics`` module has changed. The data type of has
```

Comment:
It's not clear what you mean by "format" here. Is it the types returned by these property attributes? If so, just state that changes have been made for consistency with the rest of `sympy.physics.mechanics` and this means changing those return types and unifying the symbols used for the generalized coordinates and generalized speeds.

Reply:
It is the type and the autonaming, that is why I choose for format.